dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/DependencyConflict.java

```diff
@@ -50,13 +54,15 @@ public DependencyPath getPathToSelectedArtifact() {
   public String toString() {
     Artifact selected = pathToSelectedArtifact.getLeaf();
     Artifact unselected = pathToUnselectedArtifact.getLeaf();
-
+    //    com.google:foo:1 (selected in the class path) does not have symbol 'java.lang.Object's
```

Comment:
I'm not sure you need the word "symbol" here. I think this can be

com.google:foo:1 (selected in the class path) does not have the method 'java.lang.Object#equals(Object)' but version 2 has it.

Reply:
Removed "the symbol"